src/components/DisplayBox/DisplayBox.js

```diff
@@ -88,7 +94,51 @@ export default class DisplayBox extends Component{
             data: {},
           });
         }
-        this.props.takeSuggestion(suggestion);
+
+        const client = FHIR.client({
+          serverUrl: this.props.ehrUrl,
+          tokenResponse: {
+            access_token: this.props.access_token.access_token,
+          },
+        });
+
+        // handle each action from the suggestion
+        suggestion.actions.forEach((a) => {
```

Comment:
it would be good to use a descriptive variable name, like "suggestion" or "action".

Reply:
Sounds good, I'll change it to action.